presto-main/src/main/java/com/facebook/presto/operator/aggregation/histogram/GroupMode.java

```diff
@@ -1,6 +1,6 @@
 package com.facebook.presto.operator.aggregation.histogram;
 
-public enum HistogramGroupByMode
+public enum GroupMode
```

Comment:
sure, i get legacy, but new doesn't really communicate much. legacy, sure, as in "don't use this".

i guess if it's binary, legacy/new makes sense

Reply:
also i find this confusing--it's not the implemtation of single

HistogramGroupImplementation? the single uses the old mode
and may evolve differently since 1 group vs many groups changes the data structure

Reply:
done